daemon/src/db.rs

```diff
@@ -389,29 +389,34 @@ mod tests {
         pool.acquire().await.unwrap()
     }
 
-    impl Cfd {
-        fn dummy() -> Self {
-            Self::new(
-                OrderId::default(),
-                Position::Long,
-                Price::new(dec!(60_000)).unwrap(),
-                Leverage::new(2).unwrap(),
-                Duration::hours(24),
-                Role::Taker,
-                Usd::new(dec!(1_000)),
-                "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
-                    .parse()
-                    .unwrap(),
-                OpeningFee::new(Amount::from_sat(2000)),
-                FundingRate::default(),
-                TxFeeRate::default(),
-            )
-        }
+    fn dummy_cfd() -> Cfd {
+        Cfd::new(
+            OrderId::default(),
+            Position::Long,
+            Price::new(dec!(60_000)).unwrap(),
+            Leverage::new(2).unwrap(),
+            Duration::hours(24),
+            Role::Taker,
+            Usd::new(dec!(1_000)),
+            "FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
+                .parse()
+                .unwrap(),
+            OpeningFee::new(Amount::from_sat(2000)),
+            FundingRate::default(),
+            TxFeeRate::default(),
+        )
+    }
 
+    #[async_trait::async_trait]
```

Comment:
Separate commit?

Reply:
I think we could also just solve this differently by making it a free function :)

Reply:
Done. I wanted to see if I could keep a similar API with a trait, but needing `async_trait` is just stupid.